src/main/java/cc/polyfrost/oneconfig/config/core/ConfigUtils.java

```diff
@@ -131,12 +131,53 @@ public static BasicOption addOptionToPage(OptionPage page, Option option, Field
         return configOption;
     }
 
+    public static BasicOption addOptionToPage(OptionPage page, Option option, Field field, Object instance, String category, @Nullable Migrator migrator) {
```

Comment:
if this is a public method, document it as such

Reply:
where and how